src/util.lisp

```diff
@@ -81,7 +78,14 @@
           ((vector (unsigned-byte 8))
            (length body))))))
 
+;; cl-isaac supports ISAAC-64 solely for implementations with x86-64
+;; capabilities. Use whichever-best supported capability
+(defparameter *isaac-ctx*
+  (isaac:init-self-seed :count 5
+                        :is64 #+:X86-64 t #-:X86-64 nil))
+
 (defun generate-random-id ()
   "Generates a random token."
-  (ironclad:byte-array-to-hex-string
-    (ironclad:random-data 20)))
+  (format nil "~64,'0x" (#+:X86-64 isaac:rand-bits-64
+                         #-:X86-64 isaac:rand-bits-32
```

Comment:
Good catch, fixed!

> And also, it would be great to treat ARM64 as 64bit.

cl-isaac only supports 64-bit when the `x86-64` feature is flagged: https://github.com/thephoeron/cl-isaac/blob/260cb349d4251d838d6cb6c7bda66177f7a75480/cl-isaac.asd#L29